README.md

```diff
@@ -32,14 +32,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+A programming object that represents html elements of a webpage. 
```

Comment:
Yes and no.

DOM stands for Document Object Model. 

It's an API (Application Programming Interface) that provides access to the structure of a document and a way to manipulate it.

Another way to describe that would be a way to programmatically manipulate the html elements of a webpage.

Here is some information on the DOM:
https://www.w3.org/TR/DOM-Level-2-Core/introduction.html